tests/test_e2e.py

```
@@ -1,4 +1,6 @@
+from decimal import Decimal
 import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)